crates/control/src/behavior/initial.rs

```diff
@@ -5,32 +5,36 @@ use types::{
     camera_position::CameraPosition,
     filtered_game_state::FilteredGameState,
     motion_command::{HeadMotion, ImageRegionTarget, MotionCommand},
+    primary_state::PrimaryState,
     world_state::WorldState,
 };
 
 pub fn execute(
     world_state: &WorldState,
-    expected_referee_position: Point2<Ground>,
+    expected_referee_position: Option<&Point2<Ground>>,
     pixel_target: ImageRegionTarget,
 ) -> Option<MotionCommand> {
     let filtered_game_controller_state = world_state.filtered_game_controller_state?;
+    let expected_referee_position = expected_referee_position?;
```

Comment:
Please keep also keep the old behavior, i.e.
```
    match world_state.robot.primary_state {
        PrimaryState::Initial => Some(MotionCommand::Initial),
        _ => None,
    }
```

Reply:
Done